database/inject/InjectionManager.py

```diff
@@ -21,10 +23,17 @@ async def inject(self, replay, session):
         :param session: Database session supporting flush, commit and rollback:
         """
 
+
         ## constuct and attach hashmap of events w/event.name
         self._prepare(replay)
 
         try:
+            exists = select(info).where(info.filehash == replay.filehash)
+            result = await session.execute(exists)
+            if result.scalar():
```

Comment:
quick-fix -> result fetch rows. Otherwise, always true as execution object is 'truthy'